wasm.js

```diff
@@ -2403,8 +2417,8 @@ async function execve(url, args, envs) {
   console.log('execve: ' + url + ' [' + args + '] [' + envs + ']');
   var start;
   try {
-    const instance = await fetchAndInstantiate(url, importObject);
     start = performance.now();
+    const instance = await fetchAndInstantiate(url, importObject);
```

Comment:
I had this before the perf measurement so that we could measure the actual running time of the program, not including the fetch / parse time. If we'd like to measure the fetch / parse time too, I think that should be a separate measurement, not included.